### DISCUSSION TITLE
Different dependencies than micromamba?

### DISCUSSION THREAD

**Discussion by davidbrochart** [prefix-dev/pixi#280] [Q&A] [answered]
Just of curiosity, with `micromamba`, installing `python` also pulls `pip`, while it's not the case with `pixi`.
Since I guess it's the same conda-forge dependency system under the hood, I was wondering why they behave differently. Is it because of the solver?

**Comment by wolfv** [comment, marked as answer]
That is actually a special case – the Python package in conda-forge and defaults _does not_ depend on pip. There is a configuration value for conda/mamba (https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#add-pip-as-python-dependency-add-pip-as-python-dependency) to dynamically add pip as a python dependency.

We did add it to rattler create (https://github.com/mamba-org/rattler/blob/9591c1b020bf4bd60a6082efcd1e2c06fc710ad5/crates/rattler-bin/src/commands/create.rs#L160-L169) so we could enable it relatively easily in pixi, too, but we need to decide on it :)
